python/cisa_risk_meter/build_cisa_risk_meter.py

```diff
@@ -323,7 +353,8 @@ def create_risk_meter(base_url, headers, risk_meter_name, cisa_custom_field_id):
                 cve_vuln_id = cve_vuln['id']
                 logging.info(f"Add Vuln ID {cve_vuln_id} to custom field update list")
                 vulns_to_update.append(cve_vuln_id)
-                if len(vulns_to_update) > VULN_UPDATE_LIMIT:
+                if len(vulns_to_update) >= VULN_UPDATE_LIMIT:
+                    print("")
```

Comment:
Just curious: is it just a spacer line?

Reply:
Yes, because of the "." being printed with without linefeed and I want the next output to be on a new line.